test_scripts/centos/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM centos:latest
+FROM centos:8
 
 COPY . /app
-RUN yum -y install initscripts && yum clean all
+RUN yum update && yum -y install initscripts procps which && yum clean all
```

Comment:
I think normally there's a rm -rf at the end to remove the cache that clean always forgets to but this is fine... 👍

Reply:
Yup.. added